eng/Analyzers.props

```diff
@@ -3,9 +3,10 @@
     <CodeAnalysisRuleset>$(MSBuildThisFileDirectory)CodeAnalysis.ruleset</CodeAnalysisRuleset>
   </PropertyGroup>
   <ItemGroup Condition="'$(EnableAnalyzers)' == 'true'">
-    <PackageReference Include="Microsoft.DotNet.CodeAnalysis" Version="$(MicrosoftDotNetCodeAnalysisVersion)" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="3.4.0-beta2-final" />
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.6" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.66" />
+    <PackageReference Include="Microsoft.DotNet.CodeAnalysis" Version="$(MicrosoftDotNetCodeAnalysisVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
+    <!-- Microsoft.CodeAnalysis.Common brings in external dependencies that CoreLib can't deal with. -->
```

Comment:
What is the `Microsoft.CodeAnalysis.Common` package for? What is not going to work if we remove from here?

Reply:
That package was first introduced with https://github.com/dotnet/corefx/commit/87873f55d4d0d4f2b073e544f1295761976beca8#diff-c9ed7fc6ec49880569e6c2eb08120fc4R27. I'm not exactly sure, its description even says it shouldn't be referenced directly. I'll take a closer look.